yarn.lock

```diff
@@ -13785,13 +13790,14 @@ rrweb-snapshot@^0.7.26:
   resolved "https://registry.yarnpkg.com/rrweb-snapshot/-/rrweb-snapshot-0.7.26.tgz#339769cdcfc53db4fec279501706ff6129671cb2"
   integrity sha512-z4k6swLNI5YPDFFcYp19Ggo2kE12cWG1OZCanlaMvwSOi82OOZ229ckYBCPYHbYQnCwuBF5KSMZi2i0CwpWHng==
 
-rrweb@^0.7.25, rrweb@^0.7.30:
-  version "0.7.30"
-  resolved "https://registry.yarnpkg.com/rrweb/-/rrweb-0.7.30.tgz#5a2ba3e667b5f663fc207574188fcfcd450467df"
-  integrity sha512-ODA0W2cas5yoU1jLO/UVNUrr99jIfv5nQ7jrD7gfFG09O0YaMV9I9p5SkAztQe6gv2QvxgFsyFxzWjpfD7btvw==
+rrweb@^0.7.25, rrweb@^0.7.33:
```

Comment:
hrm is this maybe because of rrweb-player or something? whys it end up w/ two versions in here

Reply:
ya `rrweb-player`